### PR TITLE
Add a new page for service accounts

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

Add a new topic for Service accounts
Removed existing info on service accounts from the Repositories topic

### CODE REVIEW

**Comment by usha-mandya** [comment]
@jeremy-castile PTAL

File: docker-hub/service-accounts.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+To create a new service account for your Team account:
+
+1. Create a new Docker ID.
+2. Create a [team](orgs.md#create-a-team) in your organization and grant it read-only access to your private repositories.
+3. Add the new Docker ID to your [organization](orgs.md#working-with-organizations).
+4. Add the new Docker ID  to the [team](orgs.md#add-a-member-to-a-team) you created earlier.
+5. Create a new [personal access token (PAT)](/access-tokens.md) from the user account and use it for CI.
+
+To create a new service account for your Pro account:
```
Comment (nebuk89) [inline docker-hub/service-accounts.md:44]: Let's remove the Pro account section for now and make the service account discussion just about teams.

**Review by nebuk89 — DISMISSED** [review]
One change just getting ready for read only tokens and to remove confusion

File: docker-hub/service-accounts.md (hunk 1 above)
Comment (usha-mandya) [reply docker-hub/service-accounts.md]: Sounds good. I've removed section on Pro accounts

File: docker-hub/service-accounts.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+1. Create a new Docker ID.
+2. Create a [team](orgs.md#create-a-team) in your organization and grant it read-only access to your private repositories.
+3. Add the new Docker ID to your [organization](orgs.md#working-with-organizations).
+4. Add the new Docker ID  to the [team](orgs.md#add-a-member-to-a-team) you created earlier.
+5. Create a new [personal access token (PAT)](/access-tokens.md) from the user account and use it for CI.
+
+> **Note**
+>
+> If you want a read-only PAT just for your open source repos, or to access
```
Comment (jerae-duffin) [inline docker-hub/service-accounts.md:46]: open-source

Comment (usha-mandya) [reply docker-hub/service-accounts.md]: Yes, that makes sense. Updated.

**Review by jerae-duffin — APPROVED** [review]
LGTM

**Review by nebuk89 — COMMENTED** [review]
LGTM

**Pull request merged**